src/Ports.elm

```diff
@@ -0,0 +1,9 @@
+port module Ports exposing (..)
```

Comment:
We definitely want to minimize any entry points into the app. We might be able to write some of these in pure elm.